sql/Procedures.sql

```diff
@@ -0,0 +1,127 @@
+DROP procedure IF EXISTS `edit_password`;
+DELIMITER $$
+CREATE DEFINER=`root`@`localhost` PROCEDURE `edit_password`( user_id int,  new_password varchar(512))
+BEGIN
+update users set password=new_password where id=user_id;
+END$$
+DELIMITER ;
+
+DROP procedure IF EXISTS `search_for_company`;
+DELIMITER $$
+CREATE DEFINER=`root`@`localhost` PROCEDURE `search_for_company`(name_search varchar(50) )
+BEGIN
+select * from companies where name=name_search;
```

Comment:
As a convention, SQL key words should be uppercase (e.g., `SELECT * FROM ... WHERE ...`)